libs/langchain-mcp-adapters/package.json

```
@@ -0,0 +1,119 @@
+{
+  "name": "@langchain/mcp-adapters",
+  "version": "0.4.5",
+  "description": "LangChain.js adapters for Model Context Protocol (MCP)",
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
+  "type": "module",
+  "packageManager": "yarn@3.5.1",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/langchain-ai/langchainjs-mcp-adapters.git"
+  },
+  "homepage": "https://github.com/langchain-ai/langchainjs-mcp-adapters#readme",
+  "bugs": {
+    "url": "https://github.com/langchain-ai/langchainjs-mcp-adapters/issues"
```

i think these are correct, double check
```suggestion
    "url": "git+https://github.com/langchain-ai/langchainjs/libs/langchain-mcp-adapters.git"
  },
  "homepage": "https://github.com/langchain-ai/langchainjs/blob/main/libs/langchain-mcp-adapters/README.md",
  "bugs": {
    "url": "https://github.com/langchain-ai/langchainjs/libs/langchain-mcp-adapters/issues"
```